packages/neuron-wallet/.prettierrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  tabWidth: 2,
+  useTabs: false,
+  printWidth: 120,
+  // TODO: This is the previous legacy configuration, should be set to es5, considering that more files are affected, and will be dealt with later.
+  // https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8
+  trailingComma: 'none',
```

Comment:
Can it automatically fix it through the command?
I see that you have removed many commas at the last of the object or array.

Reply:
It can be handled automatically, but it is likely to bring about a large number of file changes, which may cause trouble for rebasing on the subsequent large branch. Therefore, it is only marked as a TODO for now.

Reply:
Anytime it will bring about a large number of file changes, but now you set `none` will add more changes for next time.

Reply:
You're right. Initially, I thought it would be best to modify it after merging the two large branches. However, now I realize that I should first test and determine the most appropriate workflow. Once I have completed the testing, I will reply with the results.

Reply:
I have decided to directly adjust trailingComma to es5. It has been updated accordingly.